File: library/src/main/java/com/getstream/sdk/chat/logger/StreamChatSilentLogger.java
package com.getstream.sdk.chat.logger;

import android.util.Log;

import androidx.annotation.NonNull;

public class StreamChatSilentLogger implements StreamLogger {

    @Override
    public void logI(@NonNull Object tag, @NonNull String message) {
        // unused
    }

    @Override
    public void logD(@NonNull Object tag, @NonNull String message) {
        // unused
    }

    @Override
    public void logW(@NonNull Object tag, @NonNull String message) {
        // unused
    }

    @Override
    public void logE(@NonNull Object tag, @NonNull String message) {
        // unused
    }

    @Override
    public void logT(@NonNull Throwable throwable) {
        // unused
    }

    @Override
    public void logT(@NonNull Object tag, @NonNull Throwable throwable) {
        // unused
    }
}

File: app/src/main/java/io/getstream/chat/android/core/poc/library/logger/StreamChatSilentLogger.kt
package io.getstream.chat.android.core.poc.library.logger

class StreamChatSilentLogger : StreamLogger {
    override fun logT(throwable: Throwable) {
        // unused
    }

    override fun logT(tag: Any, throwable: Throwable) {
        // unused
    }

    override fun logI(tag: Any, message: String) {
        // unused
    }

    override fun logD(tag: Any, message: String) {
        // unused
    }

    override fun logW(tag: Any, message: String) {
        // unused
    }

    override fun logE(tag: Any, message: String) {
        // unused
    }
}